src/libraries/Microsoft.PowerFx.Core/FeatureFlags.cs

```diff
@@ -7,7 +7,7 @@ namespace Microsoft.PowerFx
     /// Hosts can enable these flags to try out early features. 
     /// Any flags will eventually default to true and then get removed once the feature is finalized. 
     /// </summary>
-    internal static class FeatureFlags
+    public static class FeatureFlags
```

Comment:
Strongly disagree with this. The existence of FeatureFlags is a hack to enable Canvas not to pick up some changes immediately. It should not be considered part of our public contract, because we want to be able to remove and clean up these flags easily.

Reply:
I agree with McCall. Also this needs to be immutable once it is initialized.

Reply:
We should add some checks so that it can't be changed after initializing once.